src/Microsoft.DotNet.Tools.Compiler.Native/project.json

```diff
@@ -24,8 +24,8 @@
             "version": "1.0.0-*"
         },
       "Microsoft.DotNet.ILCompiler": "1.0.2-*",
-      "Microsoft.DotNet.ObjectWriter": "1.0.2-*",
-      "Microsoft.DotNet.RyuJit": "1.0.0-*",
+      "Microsoft.DotNet.ObjectWriter": "1.0.3-*",
```

Comment:
@brthor, I will remove these as part of my packaging PR. I'm testing it right now.